src/Downloader.ts

```diff
@@ -586,19 +591,34 @@ class Downloader {
   }
 
   private async getCompressedBody(resp: any): Promise<any> {
-    return this.isMimeTypeImage(resp.headers['content-type']) ? await imagemin.buffer(resp.data, imageminOptions) : resp.data;
+    if (isBitmapImageMimeType(resp.headers['content-type'])) {
```

Comment:
I would remove this line and just `return true` in each optim conditional block (that will be quicker to execute.

Reply:
@kelson42 I don't get this part

Reply:
lets keep it like this